docs/api.rst

```diff
@@ -866,7 +867,7 @@ API Reference
                 self.config = self.parse_config(self.config_file)
                 self.asset_type = self.message[&#34;asset_type&#34;] if &#34;asset_type&#34; in self.message else &#34;&#34;
                 self.version = self.message[&#34;version&#34;] if &#34;version&#34; in self.message else &#34;&#34;
-                self.validator_version = pkg_resources.require(&#34;stac-validator&#34;)[0].version
+                self.validator_version = importlib.metadata.distribution(&#34;stac-validator&#34;)[0].version
```

Comment:
Hi. Is the [0] index here an error. I think.

Reply:
Should be fixed now.
Sorry for the mistake.

Reply:
Thanks no worries

Reply:
Getting this error in tests - 

```package = 'stac_check.lint'

    def _get_package(package) -> ModuleType:
        """Take a package name or module object and return the module.
    
        If a name, the module is imported.  If the passed or imported module
        object is not a package, raise an exception.
        """
        if hasattr(package, '__spec__'):
            if package.__spec__.submodule_search_locations is None:
                raise TypeError('{!r} is not a package'.format(
                    package.__spec__.name))
            else:
                return package
        else:
            module = import_module(package)
            if module.__spec__.submodule_search_locations is None:
>               raise TypeError('{!r} is not a package'.format(package))
E               TypeError: 'stac_check.lint' is not a package

/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/importlib/resources.py:49: TypeError